modules/nested-elements/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -0,0 +1,61 @@
+import BaseNestedTabs from './base-nested-tabs';
```

Comment:
Rename all technical `NestedTabs` to TabsV2

Reply:
handled.